app/presenters/reservation_calendar.rb

```diff
@@ -0,0 +1,31 @@
+class ReservationCalendar < SimpleDelegator
+
+  attr_accessor :hostname
+
+  def self.to_calendar(reservation)
+    ReservationCalendar.new(reservation).as_ical
+  end
+
+  def initialize(reservation, hostname)
+    super(reservation)
+    @hostname = hostname
```

Comment:
I don't necessarily have a problem with this, but I feel like there's got to be a better way to get the hostname out of the request or something.